apps/idos-data-dashboard/src/core/idos.tsx

```diff
@@ -95,7 +95,10 @@ export const Provider = ({ children }: PropsWithChildren) => {
     if (profile) {
       // @ts-expect-error
       await _sdk.setSigner(signer.type, signer.value);
-      const _pk = (await _sdk.auth.currentUser()).publicKey;
+      const _pk = _sdk.auth.currentUser.publicKey;
+
+      console.log(_sdk.auth.currentUser);
+
```

Comment:
Accidental leftover?
```suggestion

      console.log(_sdk.auth.currentUser);

```